gui/slick/js/core.js

```diff
@@ -2505,13 +2505,24 @@ var SICKRAGE = {
                     $(spinner).empty().html(message);
                 }
             };
-            
-            $.fn.loadContainer = function(path, loadingTxt, errorTxt) {
+
+            // Check the previous status of the history table, for hidden or shown, through the data attribute
+            // data-history-toggle-hidden
+            function toggleHistoryTable() {
+                // Get previous state which was saved on the wrapper
+                var showOrHide = $('#wrapper').attr('data-history-toggle');
+                $('#historydata').collapse(showOrHide);
+            }
+
+            $.fn.loadContainer = function(path, loadingTxt, errorTxt, callback) {
                 updateSpinner(spinner, loadingTxt);
                 $(this).load(srRoot + path + ' #container', function(response, status) {
                     if (status === "error") {
                         updateSpinner(spinner, errorTxt, false);
                     }
+                    if (typeof callback !== 'undefined') {
+                        callback();
```

Comment:
I've implemented it as a callback. But that was to be able to use the loadContainer() function genericly, without requiring to pass a callback function, although it's only used by one caller.